src/Arcus.EventGrid.Tests.Integration/Publishing/EventGridPublisherClientWithTrackingEventGridEventTests.cs

```diff
@@ -124,47 +124,5 @@ private static void AssertEventGridEventForData(EventGridEvent expected, EventGr
             string actual = endpoint.ConsumerHost.GetReceivedEventOrFail(expected.Id);
             ArcusAssert.ReceivedNewCarRegisteredEvent(expected, actual);
         }
-
```

Comment:
Tests that are being removed after applying coverage reporting; interesting :)

Reply:
These tests were actually unit tests that were hiding in the integration test project. Moved them.